packages/app-content-pages/server/set-logging.spec.js

```diff
@@ -10,7 +10,13 @@ describe('Server > setLogging', function () {
 
   describe('default behaviour', function () {
     before(function () {
-      setLogging = require('./set-logging')
+      setLogging = proxyquire('./set-logging', {
+        process: {
+          env: {
+            NODE_ENV: 'development'
+          }
+        }
+      })
```

Comment:
is the descriptor still accurate for this if you have to configure the node env? Is this still default behavior?

Reply:
I think this is the default, based on a very quick look at `set-logging.js`. I'm not sure. The default webpack mode might be 'none'. https://webpack.js.org/configuration/mode/

Reply:
I've changed the wording on these, because I'm not 100% sure what 'default' means here.

Reply:
The original test assumed that the default `NODE_ENV` was 'development' or undefined. Now that the tests are being run against a production build, the default `NODE_ENV` is 'production'.